helm: Add support for mounting existing TLS root CA (#13671)
gravitational/teleport: docs/pages/setup/helm-reference/teleport-kube-agent.mdx

  One option might be to use Teleport's built-in [ACME support](./teleport-cluster.mdx#acme) or enable [cert-manager support](./teleport-cluster.mdx#highavailabilitycertmanager).
</Admonition>

## `tls`

### `existingCASecretName`

| Type | Default value |
| - | - |
| `string` | `""` |

`tls.existingCASecretName` sets the `SSL_CERT_FILE` environment variable to load a trusted CA or bundle in PEM format into Teleport pods.
This can be set to inject a root and/or intermediate CA so that Teleport can build a full trust chain on startup.

> The injected CA will be used to validate TLS communicationsx, with the Proxy Service or with upstream applications.

Reviewer: ```suggestion
The injected CA will be used for TLS communication with the Proxy Service or upstream applications.
```
